backends/tfhe-cuda-backend/cuda/src/integer/integer.cuh

```diff
@@ -99,6 +99,34 @@ host_radix_blocks_rotate_left(cudaStream_t *streams, uint32_t *gpu_indexes,
       dst, src, value, blocks_count, lwe_size);
 }
 
+// reverse the blocks in a list
+// each cuda block swaps a couple of blocks
+template <typename Torus>
+__global__ void radix_blocks_reverse_lwe(Torus *src, uint32_t blocks_count,
+                                         uint32_t lwe_size) {
+
+  size_t idx = blockIdx.x;
```

Comment:
not necessary because it launches as many cuda blocks as  (radix blocks)/2, so there is no case of getting out. I added a check before to make sure the number of radix_blocks is more than 1